main.cpp

```diff
@@ -500,9 +661,24 @@ struct ToneControl
         //3 set lower limit
         bool setLowerLimit(float lowerLimit = 0.0f);
         //return true if success, else false;
-    };
-
+        
+        int toIncrement = 0;
+
+        void incrementAndSetColors(int color11, int color22, int color33)
+        {
+            setToneColors(color11, color22, color33);
+            std::cout << "before a while" << toIncrement;
+            while(++toIncrement < 100)
```

Comment:
The way this comparison and increment expression is written makes it difficult to understand what is evaluated first
separate comparisons from math operations to improve the readability of the code
```
while( toIncrement < 100 )
{
    ++toIncrement;
    //etc
}
```
- [ ] revise